### DISCUSSION TITLE
How to see complete error using this plugin?

### DISCUSSION THREAD

**Discussion by thejosess** [hadynz/obsidian-kindle-plugin#225] [Q&A]
Hello, first of all thank you for this plugin I really love it.

I just using the plugin and I tried to sync my Clippings.txt from my Kindle but only sync 6 books and show a brief error message that said: "Error parsing" and "File already exists".

I just want to know what is the problem or if I could see a more detailed message.

Thank you for your time :)

**Comment by rotello** [comment]
i ve the same problem. sometimes trying 3-4 times works... others does not. i am waiting for a solution, too.
